movie-mlops-project/model/train.py

```
@@ -0,0 +1,139 @@
+import logging
+import os
+import sys
+import argparse
+
+import mlflow
+import mlflow.xgboost
+import pandas as pd
```

Ruff (0.12.2)

`pandas` imported but unused

Remove unused import: `pandas`

(F401)